FreeRTOS/FreeRTOS
# Fix CBMC patches.

Fix CBMC patches

Description
-----------
Patches have gotten out of sync with the code.

Test Steps
-----------
Run CBMC

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
